gBrogue Source/gBrogueCode/IO.c

```diff
@@ -2486,6 +2502,16 @@ void executeKeystroke(signed long keystroke, boolean controlKey, boolean shiftKe
                                  &teal, false);
 			}
 			break;
+	    case WARNING_PAUSE_KEY:
```

Comment:
This is the only 'feature' I added. It pauses the game when you hit a warning HP level (e.g. <40%). Can be turned on and off.